Cargo.toml

```diff
@@ -35,6 +35,7 @@ tracing = "0.1.37"
 axum07 = { version="0.7", package="axum", optional=true }
 futures-util = {version="0.3", optional=true}
 tokio-util = {version = "0.7", optional=true, features=["net","codec"]}
+tokio-vsock = {version = "0.5", optional=true}
```

Comment:
`tokio-vsock` does not seem to build on Windows, which breaks  `--all-features` build.

Somewhere else in the comments there is a snippet about how to include it conditionally.

Reply:
Strange, because diff started with suggested snippet in Cargo.toml